scripts/export.py

```diff
@@ -109,61 +103,42 @@ def nemo_export(argv):
         )
         raise e
 
-    logging.info("Model {} restored from '{}'".format(model.cfg.target, nemo_in))
+    logging.info("Model {} restored from '{}'".format(model.__class__.__name__, nemo_in))
 
     if not isinstance(model, Exportable):
-        logging.error("Your NeMo model class ({}) is not Exportable.".format(model.cfg.target))
+        logging.error("Your NeMo model class ({}) is not Exportable.".format(model.__class__.__name__))
         sys.exit(1)
-    typecheck.set_typecheck_enabled(enabled=False)
```

Comment:
Wont this affect export? Typechecking interferes with export right?

Reply:
That is done inside export now.